src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int upperBase;
+    private int lowerBase;
+    private int height;
+
+    public IsoscelesTrapezoid(int upperBase, int lowerBase, int height, String color) {
+        this.upperBase = upperBase;
+        this.lowerBase = lowerBase;
+        this.height = height;
+        this.color = color;
```

Comment:
It is bad practice to instantiate fields of parent class like this it could be confusing for developer, so it will be better to use super class constructor